packages/core/src/tests/group/group.e2e.ts

```diff
@@ -35,7 +35,7 @@ regressionTest.describe('group', () => {
 
   regressionTest('item selected', async ({ page }) => {
     await page.goto('group/basic');
-    await page.locator('.btn-expand-header').click();
+    await page.locator('.btn-expand-header .sc-ix-icon-h').click();
```

Comment:
`.sc-ix-icon-h ` is a compiler generated class you should not use it as an selector